docs/config/services/getVersion.js

```diff
@@ -4,7 +4,7 @@ var path = require('canonical-path');
 /**
  * dgService getVersion
  * @description
- * Find the current version of the bower component (or node module)
+ * Find the current version of the node module
```

Comment:
If this function is only used to get node modules (afaict it does), we can simplify the implementation and remove the second and third arguments. I think this happened already when we moved most of the bower stuff to node.